### DISCUSSION TITLE
Support for Solaredge-home-battery 48V

### DISCUSSION THREAD

**Discussion by jovana** [WillCodeForCats/solaredge-modbus-multi#608] [Q&A] [answered]
Hi,

I want to replace my existing home battery (LG CHEM 7K). I want to know if the new SolarEdge battery can work on this integration. This battery can chained (up to 5 depending on the inverter). I want to have 3. It's about this battery: https://knowledge-center.solaredge.com/sites/kc/files/se-solaredge-home-battery-48V-datasheet-eng-row.pdf

If anyone is using this type, please let me know if it can be used.

Jo

**Comment by deqw** [comment, marked as answer]
@jovana I have two of these and working with this integration.

**Reply by jovana** [reply]
This is good to know! Thanks!
